.eslintrc.js

```diff
@@ -22,81 +11,11 @@ module.exports = {
         mocha: true,
       },
       rules: {
+        '@typescript-eslint/camelcase': ['off'],
```

Comment:
This is an override for `tspec` test files. @totokaka prefers non-camelcase for added readability in certain types of tests.